app/assets/javascripts/index/directions/graphhopper.js

```diff
@@ -79,5 +80,6 @@ function GraphHopperEngine(id, vehicleType) {
   };
 }
 
-OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_bicycle", "bike"), true);
-OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_foot", "foot"), true);
+OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_car", "car", true), true);
```

Comment:
We disable contraction hierarchies for car as we currently have not implemented CH with turn restrictions. But disabling CH now falls back to [a still fast enough algorithm](https://github.com/graphhopper/graphhopper/pull/780) that supports turn restrictions. For the others vehicle profiles: there is no reason yet to not use CH.